.github/workflows/ci-unix-static.yml

```diff
@@ -35,18 +35,28 @@ jobs:
         profile: minimal
         toolchain: stable
         override: true
+    - name: Install cargo-c (linux)
+      if: matrix.os == 'ubuntu-latest'
+      env:
+        LINK: https://github.com/lu-zero/cargo-c/releases/latest/download
+        CARGO_C_FILE: cargo-c-x86_64-unknown-linux-musl.tar.gz
+      run: |
+        curl -L $LINK/$CARGO_C_FILE | tar xz -C ~/.cargo/bin
+    - name: Install cargo-c (mac)
+      if: matrix.os == 'macos-latest'
+      env:
+        LINK: https://github.com/lu-zero/cargo-c/releases/latest/download
+        CARGO_C_FILE: cargo-c-macos.zip
+      run: |
+        curl -sLo $CARGO_C_FILE $LINK/$CARGO_C_FILE
+        unzip -o $CARGO_C_FILE -d ~/.cargo/bin
+        rm $CARGO_C_FILE
 
-    - name: Cache external dependencies
-      id: cache-ext
-      uses: actions/cache@13aacd865c20de90d75de3b17ebe84f7a17d57d2 # v4.0.0
-      with:
-        path: ext
-        key: ${{ runner.os }}-${{ matrix.build-type }}-${{ hashFiles('ext/*.cmd', 'ext/svt.sh') }}
     - name: Setup cmake
       uses: jwlawson/actions-setup-cmake@d06b37b47cfd043ec794ffa3e40e0b6b5858a7ec # v1.14.2
       with:
-        # CMake version 3.17 is required to build libwebp (which libsharpyuv is part of) on macOS.
-        cmake-version: '3.17.x'
+        # CMake version 3.18 is required to build libxml2
```

Comment:
no need to bump here, libxml2 is not built in this CI (it is not set as a CMake argument)